usr/share/rear/lib/layout-functions.sh

```diff
@@ -745,15 +745,15 @@ function get_part_device_name_format() {
     echo "$part_name"
 }
 #
-# apply-mappings function migrate disk device reference from an old system and
+# apply_layout_mappings function migrate disk device reference from an old system and
 # replace them with new one (from current system).
 # the relationship between OLD and NEW device is provided by $MAPPING_FILE
 # (usually disk_mappings file in $VAR_DIR).
-function apply-mappings() {
+function apply_layout_mappings() {
 
-    # apply-mappings need one argument (file which contains disk device to migrate).
+    # apply_layout_mappings need one argument (file which contains disk device to migrate).
     if [ -z $1 ] ; then
```

Comment:
Since the argument has to be a non-empty file I would suggest to use `-s "$1"` instead. As usual, this also needs to be quoted to protect against `$1` containing blanks.

Reply:
True